WebService/v1/Models/SimulationApiModel/SimulationApiModel.cs

```diff
@@ -292,5 +302,27 @@ private static SimulationStatistics GetSimulationStatistics(Simulation simulatio
 
             return statistics;
         }
+
+        private IRateLimitingConfig GetRateLimits(IRateLimitingConfig defaultRateLimits)
```

Comment:
can you move this into `RateLimitingConfig.ToServiceModel()` ?

Reply:
updated